pkg/storage/etcd/etcd_watcher.go

```diff
@@ -359,9 +359,6 @@ func (w *etcdWatcher) sendAdd(res *etcd.Response) {
 		return
 	}
 	action := watch.Added
-	if res.Node.ModifiedIndex != res.Node.CreatedIndex {
```

Comment:
sendAdd() is called many more places than the narrow use case discussed... what side-effects will this have elsewhere?